Usage test failed with the following error:
```
1> [CMake] CMake Error at F:/PRreview/vcpkg/installed/x64-windows/share/SDL3_ttf/SDL3_ttf-shared-targets.cmake:60 (set_target_properties):
1> [CMake]   The link interface of target "SDL3_ttf::SDL3_ttf-shared" contains:
1> [CMake] 
1> [CMake]     SDL3::Headers
1> [CMake] 
1> [CMake]   but the target was not found.
```

--- a/ports/sdl3-ttf/portfile.cmake
+++ b/ports/sdl3-ttf/portfile.cmake
@@ -9,7 +9,7 @@ vcpkg_from_github(
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     FEATURES
         harfbuzz SDLTTF_HARFBUZZ
-	svg SDLTTF_PLUTOSVG
+	svg      SDLTTF_PLUTOSVG
 )
 
 vcpkg_cmake_configure(
@@ -22,9 +22,9 @@ vcpkg_cmake_configure(
 
 vcpkg_cmake_install()
 if(EXISTS "${CURRENT_PACKAGES_DIR}/cmake")
-    vcpkg_cmake_config_fixup(PACKAGE_NAME SDL3_ttf CONFIG_PATH cmake)
+    vcpkg_cmake_config_fixup(PACKAGE_NAME sdl3_ttf CONFIG_PATH cmake)
 else()
-    vcpkg_cmake_config_fixup(PACKAGE_NAME SDL3_ttf CONFIG_PATH lib/cmake/SDL3_ttf)
+    vcpkg_cmake_config_fixup(PACKAGE_NAME sdl3_ttf CONFIG_PATH lib/cmake/SDL3_ttf)
 endif()
 
 vcpkg_fixup_pkgconfig()

--- a/ports/sdl3-ttf/portfile.cmake
+++ b/ports/sdl3-ttf/portfile.cmake
@@ -27,6 +27,12 @@ else()
     vcpkg_cmake_config_fixup(PACKAGE_NAME sdl3_ttf CONFIG_PATH lib/cmake/SDL3_ttf)
 endif()
 
+vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/share/sdl3_ttf/SDL3_ttfConfig.cmake"
+"# sdl3_ttf cmake project-config input for CMakeLists.txt script"
+[[# sdl3_ttf cmake project-config input for CMakeLists.txt script
+include(CMakeFindDependencyMacro)
+find_dependency(SDL3 CONFIG)]])
+
 vcpkg_fixup_pkgconfig()
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")

--- a/ports/sdl3-ttf/usage
+++ b/ports/sdl3-ttf/usage
@@ -1,4 +1,4 @@
 sdl3-ttf provides CMake targets:
 
-    find_package(SDL3_ttf CONFIG REQUIRED)
-    target_link_libraries(main PRIVATE $<IF:$<TARGET_EXISTS:SDL3_ttf::SDL3_ttf>,SDL3_ttf::SDL3_ttf,SDL3_ttf::SDL3_ttf-static>)
+  find_package(SDL3_ttf CONFIG REQUIRED)
+  target_link_libraries(main PRIVATE $<IF:$<TARGET_EXISTS:SDL3_ttf::SDL3_ttf>,SDL3_ttf::SDL3_ttf,SDL3_ttf::SDL3_ttf-static>)

--- a/ports/sdl3-ttf/vcpkg.json
+++ b/ports/sdl3-ttf/vcpkg.json
@@ -5,7 +5,10 @@
   "homepage": "https://www.libsdl.org/projects/SDL_ttf/",
   "license": "Zlib",
   "dependencies": [
-    "freetype",
+    {
+      "name": "freetype",
+      "default-features": false
+    },
     {
       "name": "sdl3",
       "default-features": false

--- a/versions/s-/sdl3-ttf.json
+++ b/versions/s-/sdl3-ttf.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "2dbeb0be407e288701632522c8d735ceadbe97bc",
+      "git-tree": "5f357b9177182cd52a9ea83779ff6e2ce0a5eddc",
       "version": "3.1.0",
       "port-version": 0
     }

--- a/versions/s-/sdl3-ttf.json
+++ b/versions/s-/sdl3-ttf.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "5f357b9177182cd52a9ea83779ff6e2ce0a5eddc",
+      "git-tree": "580095b6cdeb525fe480869f54f801b064fbccd7",
       "version": "3.1.0",
       "port-version": 0
     }

--- a/versions/s-/sdl3-ttf.json
+++ b/versions/s-/sdl3-ttf.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "580095b6cdeb525fe480869f54f801b064fbccd7",
+      "git-tree": "fa0239502489d787a074de617542309b0cfb7e6e",
       "version": "3.1.0",
       "port-version": 0
     }

